### PR TITLE
build MPFR in GCC stage 1 without LTO if (system) GCC used is too old

### PR DESCRIPTION
**Pull request by boegel**
fix for https://github.com/easybuilders/easybuild-easyconfigs/issues/6374

### CODE REVIEW

**Comment by boegel** [comment]
@migueldiascosta Please test this for building `GCCcore` 7.3.0 on CentOS 6.x?

**Comment by boegel** [comment]
2nd try, please test @migueldiascosta?

File: easybuild/easyblocks/g/gcc.py [hunk 1]
```diff
@@ -120,6 +120,47 @@ def create_dir(self, dirname):
         except OSError, err:
             raise EasyBuildError("Can't use dir %s to build in: %s", dirpath, err)
 
+    def disable_lto_mpfr_old_gcc(self, objdir):
+        """
+        # if GCC version used to build stage 1 is too old, build GCC without LTO support in stage 1
```
Comment (migueldiascosta) [inline easybuild/easyblocks/g/gcc.py]: update comment to build `MPFR`, instead of `GCC`?

**Comment by migueldiascosta** [comment]
Successful test reports on CentOS 6.9 for 
- 7.3.0 at https://gist.github.com/migueldiascosta/8e9de683983e08eec9884143faa38337
- 8.1.0 at https://gist.github.com/migueldiascosta/ffe8ed6bf45ba872c388168934d0d589

**Comment by boegel** [comment]
Also tested on CentOS 7 with `GCCcore/4.9.3`, `GCCcore/6.4.0`, `GCCcore/7.3.0` and `GCCcore/8.1.0`, works like a charm, so good to go (once Travis approves)?

**Comment by bartoldeman** [comment]
lgtm, happy that the suggestion worked out alright.

**Pull request merged**